app/models/order.rb

```diff
@@ -18,15 +18,25 @@ def set_defaults
     self.state = "Created"
   end
 
-  def finalize_order
+  def transition_state
```

Comment:
@lindgrenj6 Should we move this into a Service Object, because this is business logic and Jason had said he wanted the app/models to be just models. This might be opportunity to fix that

Reply:
Yes I like that idea especially if that's what we're supposed to be doing. I got it moved out into a new service class, `Catalog::OrderStateTransition`.